### PR TITLE
Care: Fixed problem with missing columns

### PR DESCRIPTION
**Pull request by kkrampa**
@czue They start training on Thursday so it would be good to have this merged today.

### CODE REVIEW

File: custom/care_pathways/sqldata.py [hunk 1]
```diff
@@ -1,10 +1,14 @@
+from collections import OrderedDict
```
Comment (dimagimon) [inline custom/care_pathways/sqldata.py]: F401 'OrderedDict' imported but unused

**Comment by czue** [comment]
:+1: on :white_check_mark:

**Comment by czue** [comment]
@kkrampa sorry looks like this missed deploy. let me know if you think necessary to do an emergency deploy today so it can be tested before tomorrow

**Comment by kkrampa** [comment]
@czue It would be good to have Jennifer and Molly opinion before tomorrow's training so if it's not a huge problem deploy will be very helpful

**Comment by kkrampa** [comment]
@snopoke @czue Thanks for deploy!

**Pull request merged**